### ISSUE TITLE
Unreachable statement when using panic at the end of function

### ISSUE THREAD

**Issue by qarmin** [stevenliebregt/fun_time#7]
```
warning: unreachable statement
  --> czkawka_gui/src/language_functions.rs:73:1
   |
73 | #[fun_time(message = "ABC")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ unreachable statement
...
81 |     panic!("Not found proper text");
   |     ------------------------------- any code following this expression is unreachable
   |
   = note: this warning originates in the attribute macro `fun_time` (in Nightly builds, run with -Z macro-backtrace for more info)
```
inside
```
#[fun_time(message = "ABC")]
pub fn get_language_from_combo_box_text(combo_box_text: &str) -> Language {
    for lang in LANGUAGES_ALL {
        if lang.combo_box_text == combo_box_text {
            return lang;
        }
    }

    panic!("Not found proper text");
}
```

**Comment by stevenliebregt** [comment]
Yes, that is most probably unavoidable as it needs to fetch the elapsed time at the end of the function block. What the macro expands to is basically:

```rust
let start_time = std::time::Instant::now();

let original_return_value = <your function content here>

let elapsed = start_time.elapsed();
```

This means that if your function panics, the elapsed() call is indeed unreachable.

If you know a way around this I'm happy to integrate this.

**Comment by stevenliebregt** [comment]
Fixed in `0.3.2` https://github.com/stevenliebregt/fun_time/releases/tag/0.3.2
